src/Dockerfile.fedora

```diff
@@ -1,4 +1,4 @@
-FROM registry.fedoraproject.org/{{ spec.latest_fedora }}/s2i-core:latest
```

Comment:
What about to use at least one tag from https://registry.fedoraproject.org/repo/f31/s2i-core/tags/
s2i-core w/o latest is really general.

Reply:
This was just a blind try to work around the weird CI failure:
```
received unexpected HTTP status: 503 Service Temporarily Unavailable
```

But seems like tags do not have an effect on this